### PR TITLE
DOMA-4067 fix contact duplicates

### PR DESCRIPTION
**Pull request by nomerdvadcatpyat**
It is necessary to make it so that it is impossible to create contacts with the same phone numbers on one unitName.
Made a constraint on the uniqueness of property, unitName, unitType and phone.
Wrote a migration that removes duplicate contacts and relinks contacts in tickets and meterReadings

### CODE REVIEW

File: apps/condo/domains/ticket/schema/CreateResidentTicketService.test.js [hunk 1]
```diff
@@ -110,25 +110,6 @@ describe('CreateResidentTicketService', () => {
         expect(contact.id).toMatch(UUID_RE)
     })
 
-    test('resident: do not create contact when create resident ticket with contact with same data', async () => {
```
Comment (pahaz) [inline apps/condo/domains/ticket/schema/CreateResidentTicketService.test.js:113]: why you want to remove this test?

File: apps/condo/domains/contact/schema/Contact.js [hunk 2]
```diff
@@ -159,6 +158,16 @@ const Contact = new GQLListSchema('Contact', {
             }
         },
     },
+    kmigratorOptions: {
+        constraints: [
+            {
+                type: 'models.UniqueConstraint',
+                fields: ['property', 'unitName', 'unitType', 'phone'],
```
Comment (pahaz) [inline apps/condo/domains/contact/schema/Contact.js:165]: probably, we need to add some test for that case

**Review by pahaz — APPROVED** [review]

File: apps/condo/domains/ticket/schema/CreateResidentTicketService.test.js (hunk 1 above)
Comment (nomerdvadcatpyat) [reply apps/condo/domains/ticket/schema/CreateResidentTicketService.test.js]: `CreateResidentTicket` is a deprecated service, I removed it in another pr

File: apps/condo/domains/contact/schema/Contact.js (hunk 2 above)
Comment (nomerdvadcatpyat) [reply apps/condo/domains/contact/schema/Contact.js:165]: we have such tests in `Contact.test.js` in `unique constraint` describe block

**Review by akarjakin — APPROVED** [review]

**Pull request merged**